.github/workflows/testxpr.yml

```diff
@@ -0,0 +1,137 @@
+name: hdf5 TestExpress CI
+
+# Run TestExpress CI daily at 04:00 CDT (9:00 UTC)
+on:
+  workflow_dispatch:
+  schedule:
```

Comment:
Since this will mostly run on a schedule, we may want to add a status link in the README or somewhere else so we can monitor it.

Reply:
Maybe, sounds good.